kong/db/declarative/init.lua

```diff
@@ -356,7 +363,13 @@ function declarative.load_into_cache(entities, hash, shadow_page)
 
       for _, unique in ipairs(uniques) do
         if item[unique] then
-          local cache_key = entity_name .. "|" .. unique .. ":" .. item[unique]
+          local unique_key = item[unique]
+          if type(unique_key) == "table" then
+            local _
+            _, unique_key = next(unique_key)
```

Comment:
Wondering if we should add a comment here like
```
-- this assumes that foreign keys are not composite
```
but I guess various other places in the DAO make that assumption currently already.

Reply:
Added that comment.